services/order.py

```diff
@@ -0,0 +1,37 @@
+from django.db import transaction, IntegrityError
+from db.models import Order, Ticket, User, MovieSession
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str | None = None
+) -> list[dict]:
+    try:
+        user = User.objects.get(username=username)
+        order = Order.objects.create(user=user)
+        if date:
+            order.created_at = date
+            order.save()
+
+        for ticket_data in tickets:
+            movie_session = MovieSession.objects.get(
+                id=ticket_data["movie_session"]
+            )
+            Ticket.objects.create(
+                movie_session=movie_session,
```

Comment:
you can pass movie_session_id here without explicitly creating a movie session object